cases/src/org/commcare/cases/ledger/instance/LedgerInstanceTreeElement.java

```diff
@@ -93,7 +93,7 @@ public void attachStringCache(Interner<String> stringCache) {
      */
     public LedgerChildElement getChild(String name, int multiplicity) {
         if (multiplicity == TreeReference.INDEX_TEMPLATE) {
```

Comment:
We should be checking thiis for both name and multiplicity.

IE: we only wanna return the ledger/case template if the request on templates was for a "ledger" template, not a "test" tempalte